### ISSUE TITLE
Potential vulnerability in Puppet file: Admin by default

### ISSUE THREAD

**Issue by cantebigene** [pseiler/puppet-lanserver#4]
The following script seems to have an admin by default configuration `admin_user=admin`: https://github.com/pseiler/puppet-lanserver/blob/1018dafa82101a0ae41d507ec64535df5aca148c/lanserver/manifests/params.pp#L142

Specifying default users as administrative users gives a level of privilege to users that only an admin should have.

This type of smell can lead to a well-known type of vulnerability, as documented by CWE ([CWE-250](https://cwe.mitre.org/data/definitions/250.html)). Admin accounts can be exploited to gain access to sensitive data and execute unauthorized code or commands.

Setting user accounts as admin by default in **Puppet** configuration files is considered to be a security smell (cf. [[icse20](https://dl.acm.org/doi/pdf/10.1109/ICSE.2019.00033)]).

**Recommendation**
Developers should not give admin passwords and usernames to any user accounts. User accounts should have the minimum possible security privilege and those accounts are the ones that should be used as default instead of admin accounts.

**Comment by pseiler** [comment]
Yeah this is a good idea. I also need to merge all password fields into one with a dedicated user for it.
This will be fixed soon. Thank you!
